spec/vcr/applications_controller/show/sale_application.yml

```diff
@@ -8,7 +8,7 @@ http_interactions:
       string: grant_type=password&client_id=<<SALESFORCE_CLIENT_ID>>&client_secret=<<SALESFORCE_CLIENT_SECRET>>&username=<<SALESFORCE_USERNAME>>&password=<<SALESFORCE_PASSWORD>><<SALESFORCE_SECURITY_TOKEN>>
     headers:
       User-Agent:
-      - Faraday v0.15.4
+      - Faraday v0.17.3
```

Comment:
wondering if these cassette updates were to try to keep things up-to-date or for some other reason.

Reply:
I think some of these might be by accident while I was still learning how VCR cassettes worked. I can verify the cassette changes and remove unnecessary ones before I merge

Reply:
it's OK! we did enough QA that I feel OK with the snapshot and vcr changes.